pkg/helpers/newapp/app/imageref_test.go

```diff
@@ -89,6 +89,67 @@ func TestBuildConfigOutput(t *testing.T) {
 	}
 }
 
+func TestImageRefAsImageStreamSuggestedNameContainsSlashes(t *testing.T) {
```

Comment:
```suggestion
func TestSuggestName(t *testing.T) {
```
This test and the below `TestImageStreamRefNaming` test seem like they should be combined since they are both testing the same function.